src/main/java/org/primeframework/mvc/security/cookiesession/SerializedSessionContainer.java

```diff
@@ -0,0 +1,24 @@
+/*
+ * Copyright (c) 2024, Inversoft Inc., All Rights Reserved
+ */
+package org.primeframework.mvc.security.cookiesession;
+
+import java.time.ZonedDateTime;
+import java.util.UUID;
+
+import com.fasterxml.jackson.annotation.JsonCreator;
+
+public class SerializedSessionContainer {
+  public UUID userId;
+  public String sessionId;
+  public ZonedDateTime signInInstant;
```

Comment:
- `loginInstant`
- Run the IJ formatter on this file (and others if you have not)

Reply:
Done